### DISCUSSION TITLE
Bazzite Rog Ally Controller not working

### DISCUSSION THREAD

**Discussion by penaverdeglenn** [unknownskl/xbox-xcloud-client#1326] [Q&A]
controller is not working but is detected during game mode and is placed at #2. but when running the stream its not working or responding

**Comment by RebootFixesAll** [comment]
I'm having the same issue. Did you ever find a solution?

**Comment by dfunked** [comment]
Same issue here. Works OK on Steam OS on the Ally, but not with Bazzite.
